packages/plumier/test/integration/application/return-value.spec.ts

```diff
@@ -69,7 +69,7 @@ describe("Return value", () => {
         class AnimalController {
```

Comment:
Please revert this test.. create a new test case here `packages/plumier/test/integration/application/action-result.spec.ts` at the bottom.

Reply:
You can create more appropriate test case name such as "Should able to set array value on setHeader" etc.